src/store/index.ts

```diff
@@ -0,0 +1,59 @@
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
+import { readFileSync, writeFileSync } from "fs";
+import { version as latest } from "../config";
+import zoomUsersReducer, { name as zoomUsersSlice} from "./zoom";
+import linkedAccountsReducer, { name as linkedAccountsSlice } from "./link";
+
+const PERSISTED_STATE = "../savedState.json"
```

Comment:
Two nits, both very small:
* For clarity, can we call this something that includes the word file or path? I kept thinking that this referred to the state object itself. (ie. `PERSISTED_STATE_FILE`)
* For consistency, can we `path.resolve` this relative to `__dirname` instead of the cwd? The cwd may vary.

Reply:
I've moved it into an environment variable because for local nesting it doesn't need to be a level up but in prod it does